### PR TITLE
Migration to Spine 0.8.32-SNAPSHOT

### PR DESCRIPTION
**Pull request by IlliaShepilov**
This PR resolves the issues (a compilation errors and a broken tests) occurred after Spine version update.

Summary:

- A part of the business logic fixed to pass the Spine validations.
- Setup of the tests located under `...agregate.definition` package fixed.

### CODE REVIEW

**Comment by IlliaShepilov** [comment]
@dmytro-grankin PTAL.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (armiol) [inline .idea/codeStyleSettings.xml]: Please make sure the codestyle settings are taken from the current `core-java`.

File: model/src/main/proto/todolist/c/commands.proto [hunk 2]
```diff
@@ -162,8 +162,11 @@ message CreateDraft {
     // An identifier of the target task.
     TaskId id = 1;
 
+    // A description for the draft task.
+    string description = 2 [(required) = true];
```
Comment (armiol) [inline model/src/main/proto/todolist/c/commands.proto]: We'll need to discuss that. Something isn't right here.

**Pull request merged**